tools/ci/sim_results.py

```diff
@@ -0,0 +1,66 @@
+import sqlite3
+import pandas as pd
+
+from pathlib import Path
+from dataclasses import dataclass
+from typing import Optional, Dict
+
+
+@dataclass
+class SimRecordedData:
+    # TODO: expend to all all tables as needed
```

Comment:
Possibly it is easier to have a set of SQL queries realising test assertions such as "positions are within boundaries" instead of loading all data into memory.